apps/docs/content/getting-started/releases-versioning.mdx

```diff
@@ -13,6 +13,10 @@ Unlike many JavaScript packages distributed on [NPM](https://www.npmjs.com/), th
 - Minor version bumps are released on a regular cadence - approximately monthly. **They may contain breaking changes**. We aim to make breaking changes as minimally disruptive as possible, but tldraw is actively evolving as we add new features. We recommend updating tldraw at a similar pace to our release cadence, and be sure to check the release notes.
 - Patch version bumps are for bugfixes and hotfixes that can't wait for the next cadence release.
 
+## Upcoming releases
+
+Information about upcoming changes to the tldraw SDK can be found [here](https://tldraw.notion.site/upcoming-sdk-changes).
+
```

Comment:
We spoke about this in the office - let's remove this from the docs site for now?

```suggestion
```